Feat/makefile

rvdwijngaard opened the pull request.

sergii4 commented:
Hi @rvdwijngaard, thank you for the contribution. I assume Makefile and minimal lazy vim config changes arem't connected. If so could you please separate in two different PRs?

Why do we need that lazy minimal config into the repo?

rvdwijngaard commented:
Hi @sergii4, I added that lazy minimal config because otherwise, the unit tests of this repo do not run. I have created issue #69 for this. I added the Makefile which executes the unit tests with a minimal set of dependencies. Perhaps the name of the Lua file is not well chosen. The purpose is to install Plenary with a minimal set of dependencies.
